Tests/scripts/collect_tests/utils.py

```diff
@@ -171,8 +171,13 @@ def __init__(self, path: Path):
     @property
     def _has_no_id(self):
         # some content files may not have an id
-        return self.path.name == 'pack_metadata.json' or self.path.name.endswith(
-            '_schema.json') or self.path.name.endswith('testdata.json')
+        file_path_splitted = self.path.parts
+        return self.path.name == 'pack_metadata.json' \
+            or self.path.name.endswith('_schema.json') \
+            or self.path.name.endswith('testdata.json') \
+            or len(file_path_splitted) > 1 \
+                and file_path_splitted[-2].lower().startswith('releasenotes') \
+                and file_path_splitted[-1].lower().endswith('.json')
```

Comment:
instead of `.endswith`, better chech `path.suffix == '.json'`

Reply:
https://github.com/demisto/content/pull/24090/commits/707e1d5d7aa6ab96b573f99198b9037e807d3739